doc/nrf/releases_and_maturity/releases/release-notes-2.9.0.rst

```diff
@@ -0,0 +1,836 @@
+This section provides detailed lists of changes by :ref:`sample <samples>`.
+
+Amazon Sidewalk samples
+-----------------------
+
+|no_changes_yet_note|
+
+Bluetooth samples
+-----------------
+
+* Added:
+
+  * The :ref:`channel_sounding_ras_reflector` sample demonstrating how to implement a Channel Sounding Reflector that exposes the Ranging Responder GATT Service.
+  * The :ref:`channel_sounding_ras_initiator` sample demonstrating basic distance estimation with Channel Sounding by setting up a Channel Sounding Initiator that acts as a Ranging Requestor GATT Client.
+  * The :ref:`bt_peripheral_with_multiple_identities` sample demonstrating how to use a single physical device to create and manage multiple advertisers, making it appear as multiple distinct devices by assigning each a unique identity.
+  * The :ref:`bt_scanning_while_connecting` sample demonstrating how to establish multiple connections faster using the :kconfig:option:`CONFIG_BT_SCAN_AND_INITIATE_IN_PARALLEL` Kconfig option.
+  * Support for the ``nrf54l15dk/nrf54l05/cpuapp`` and ``nrf54l15dk/nrf54l10/cpuapp`` board targets in the following samples:
+
+    * :ref:`direct_test_mode`
+    * :ref:`peripheral_hids_mouse`
+    * :ref:`peripheral_lbs`
+    * :ref:`power_profiling`
+    * :ref:`peripheral_uart`
+
+* Updated:
+
+  * Configurations of the following Bluetooth samples to make the :ref:`Zephyr Memory Storage (ZMS) <zephyr:zms_api>` the default settings backend for all board targets that use the MRAM technology:
+
+      * :ref:`bluetooth_central_hids`
+      * :ref:`peripheral_hids_keyboard`
+      * :ref:`peripheral_hids_mouse`
+      * :ref:`central_and_peripheral_hrs`
+      * :ref:`central_bas`
+      * :ref:`central_nfc_pairing`
+      * :ref:`central_uart`
+      * :ref:`peripheral_bms`
+      * :ref:`peripheral_cgms`
+      * :ref:`peripheral_cts_client`
+      * :ref:`peripheral_lbs`
+      * :ref:`peripheral_mds`
+      * :ref:`peripheral_nfc_pairing`
+      * :ref:`power_profiling`
+      * :ref:`peripheral_rscs`
+      * :ref:`peripheral_status`
+      * :ref:`peripheral_uart`
+      * :ref:`ble_rpc_host`
+
+    As a result, all :ref:`zephyr:nrf54h20dk_nrf54h20` configurations of the affected samples were migrated from the NVS settings backend to the ZMS settings backend.
+  * Testing steps in the :ref:`peripheral_hids_mouse` to provide the build configuration that is compatible with the `Bluetooth Low Energy app`_ testing tool.
+
+  * :ref:`power_profiling` sample that now has added support for the :ref:`zephyr:nrf54h20dk_nrf54h20` board target.
+
+Bluetooth Fast Pair samples
+---------------------------
+
+* :ref:`fast_pair_locator_tag` sample:
+
+  * Updated the MCUboot bootloader configuration for the :ref:`zephyr:nrf54l15dk_nrf54l15` board target to enable the :kconfig:option:`CONFIG_FPROTECT` Kconfig option that is used to protect the bootloader partition against memory corruption.
+
+Bluetooth Mesh samples
+----------------------
+
+|no_changes_yet_note|
```

Comment:
Can you add the following for mesh sample supported target change.

* Added support for nRF54L15 in the following samples:

  * :ref:`bluetooth_mesh_sensor_client` sample
  * :ref:`bluetooth_mesh_sensor_server` sample
  * :ref:`bluetooth_ble_peripheral_lbs_coex` sample
  * :ref:`bt_mesh_chat` sample
  * :ref:`bluetooth_mesh_light_switch` sample
  * :ref:`bluetooth_mesh_silvair_enocean` sample
  * :ref:`bluetooth_mesh_light_dim` sample
  * :ref:`bluetooth_mesh_light` sample
  * :ref:`ble_mesh_dfu_target` sample
  * :ref:`bluetooth_mesh_light_lc` sample
  * :ref:`ble_mesh_dfu_distributor` sample

* Added support for nRF54L05 in the following samples:

  * :ref:`bluetooth_mesh_sensor_client` sample
  * :ref:`bluetooth_mesh_sensor_server` sample
  * :ref:`bluetooth_ble_peripheral_lbs_coex` sample
  * :ref:`bt_mesh_chat` sample
  * :ref:`bluetooth_mesh_light_switch` sample
  * :ref:`bluetooth_mesh_silvair_enocean` sample
  * :ref:`bluetooth_mesh_light_dim` sample
  * :ref:`bluetooth_mesh_light` sample
  * :ref:`bluetooth_mesh_light_lc` sample

Reply:
Done.